### PR TITLE
Update mjpeg.markdown

### PR DESCRIPTION
**Pull request by pagadrift85**
I've added an example code to use a DCS-933L perfect working in HA integration.
This code works without problem for me ad in my lovelace card i see a preview jpeg snapshot in the card and if i click on it, i see the live stream!

## Proposed change

- I propose the correct code for perfect integration of a DCS-933L IP cam

## Type of change

- [ ]Correct code fot DCS-933L ip cam

## Checklist

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @pagadrift85, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17407).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/mjpeg.markdown [hunk 1]
```diff
@@ -81,3 +81,13 @@ camera:
     password: BLUE_IRIS_PASSWORD
     authentication: basic
 ```
+
+Example of using a DCS-930L Wireless N Network Camera from D-Link:
+NB: if you don't change the default port, it is not necessary to put it in the code at IP_CAM
```
Comment (frenck) [inline source/_integrations/mjpeg.markdown]: Small tweak, as the port doesn't harm, additionally, a new line.
```suggestion
Example of using a DCS-930L Wireless N Network Camera from D-Link:

```

File: source/_integrations/mjpeg.markdown [hunk 2]
```diff
@@ -81,3 +81,13 @@ camera:
...
+```yaml
+camera:
+  - platform: mjpeg
+    name: YOUR_FRIENDLY_NAME
+    still_image_url: http://USER:PASSWORD@IP_CAM:PORT/image/jpeg.cgi
+    mjpeg_url: http://USER:PASSWORD@IP_CAM:PORT/video/mjpg.cgi
```
Comment (frenck) [inline source/_integrations/mjpeg.markdown]: Updates to match or latest YAML style guide.

```suggestion
camera:
  - platform: mjpeg
    name: "YOUR_FRIENDLY_NAME"
    still_image_url: "http://USER:PASSWORD@IP_CAM:PORT/image/jpeg.cgi"
    mjpeg_url: "http://USER:PASSWORD@IP_CAM:PORT/video/mjpg.cgi"
```

**Review by frenck — APPROVED** [review]
Thanks, @pagadrift85 👍

**Comment by pagadrift85** [comment]
Thank you for approving.
I noticed another my small error sorry: at line 85 the model of ip cam is DCS-933L and not DCS-930L, you can change it?

at this point, i've to delete branch and delete this fork or not? 
Nuova immagine bitmap

**Pull request merged**